schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id int,
```

Comment:
- [ ] [OPTIONAL] Kindly make the id column not null and autoincremented as you don't need to insert it manually.
```
id int NOT NULL IDENTITY(1,1)
```
- [ ] [OPTIONAL] Kindly mark the id column as the primary key to uniquely identify each record in a table.
```
PRIMARY KEY (id)
```